app/views/ops/_label_tag_mapping_form.html.haml

```diff
@@ -0,0 +1,47 @@
+- url = url_for(:action => 'label_tag_mapping_field_changed', :id => "#{@lt_map && @lt_map.id ? @lt_map.id : "new"}")
+#main_div
+  = render :partial => "layouts/flash_msg"
+  = form_tag({:action => 'label_tag_mapping_update'},
+             :id     => "label_tag_mapping_form",
+             :class  => "form-horizontal",
+             :method => :post) do
+    - disabled = !@lt_map.nil?
```

Comment:
This might be worth a comment.
(Background: If user wants to change label name of existing mapping, the semantics would have to be completely destroying the mapping & category and creating a new mapping, so we decided to make it uneditable and have user consciously delete and create a new one.  We might revisit this in the future.)